stack.yaml

```diff
@@ -15,11 +15,6 @@ extra-deps:
     subdirs:
       - llvm-hs
       - llvm-hs-pure
-  - megaparsec-8.0.0
```

Comment:
I think those might have been pinned for performance reasons, or to work around bugs that were present in the older versions. Are they causing any issues?